cmd/agentctl/client/api.go

```diff
@@ -12,6 +12,7 @@ import (
 	"go.ligato.io/vpp-agent/v3/client"
 	"go.ligato.io/vpp-agent/v3/cmd/agentctl/api/types"
 	"go.ligato.io/vpp-agent/v3/plugins/kvscheduler/api"
+	restapi "go.ligato.io/vpp-agent/v3/plugins/restapi/types"
```

Comment:
We should avoid having two `types` packages for REST. That will just confuse people. 

Also, I would actually avoid adding more stuff to `restapi` package since it currently mostly contains handlers that should actually be defined elsewhere (e.g. telemetry handlers to telemetry plugin, vpp cli to govppmux and VPP/Linux CRUD operations should be registered in more unified way by registering them from their respective plugins..)

Let's keep all the types for REST in `../agentctl/api/types` package for now just to make it simple.